driver-core/src/main/com/mongodb/internal/connection/ExponentiallyWeightedMovingAverage.java

```diff
@@ -30,11 +27,11 @@
         this.alpha = alpha;
     }
 
-    void reset() {
+    synchronized void reset() {
```

Comment:
Since both the RTT and Server Monitor share an instance of this class, we now have to make it threadsafe

Reply:
Ack